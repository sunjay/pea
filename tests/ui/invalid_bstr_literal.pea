fn main() {
    // Invalid byte strings
    expect_error!(b"\q");
    expect_error!(b"\x");
    expect_error!(b"\x1");
    expect_error!(b"\xq");
    expect_error!(b"\x1q");
    expect_error!(b"\");
}
