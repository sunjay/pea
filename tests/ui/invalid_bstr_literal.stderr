[tests/ui/invalid_bstr_literal.pea:3:21-3:22] error: unknown character escape: `q`

[tests/ui/invalid_bstr_literal.pea:4:21-4:22] error: numeric character escape is too short

[tests/ui/invalid_bstr_literal.pea:5:21-5:23] error: numeric character escape is too short

[tests/ui/invalid_bstr_literal.pea:6:21-6:23] error: invalid character in numeric character escape: `q`

[tests/ui/invalid_bstr_literal.pea:7:21-7:24] error: invalid character in numeric character escape: `q`

[tests/ui/invalid_bstr_literal.pea:8:19-9:2] error: unterminated double quote string

error: aborting due to 6 previous errors

